JSDemos/Demos/HtmlEditor/ToolbarCustomization/Vue/App.vue

```diff
@@ -8,8 +8,9 @@
         <DxItem name="redo"/>
         <DxItem name="separator"/>
         <DxItem
-          :accepted-values="headerValues"
           name="header"
+          :accepted-values="headerValues"
+          :option="headerOptions"
```

Comment:
Shouldn't it be like this?
```suggestion
          :options="headerOptions"
```

Reply:
Fixed: [e74e9aa0894fdc88f313789f8799b626ce473904].